doc/languages-frameworks/python.section.md

```diff
@@ -186,7 +186,7 @@ building Python libraries is `buildPythonPackage`. Let's see how we can build th
 `toolz` package.
 
 ```nix
-{ # ...
+{ lib, buildPythonPackage, fetchPypi }:
```

Comment:
I think it's useful to have these to note that you don't just pass `python` to python modules.